python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -463,3 +467,10 @@ def is_c_contiguous(
             return False
         cumulative_stride *= dim
     return True
+
+
+cpdef Column column_from_column_view(Column col):
+    """
+    Return a new Column from a Column.view().
+    """
+    return Column.from_libcudf(move(make_unique[column](col.view())))
```

Comment:
Do you need this one when you can just call `col.column_from_self_view`?

Reply:
Oops this was a remnant when I thought this should have been a free function. Removed